.github/workflows/ci.yml

```diff
@@ -29,7 +29,7 @@ jobs:
       max-parallel: 4
       matrix:
         python-version: ['3.6', '3.7', '3.8', '3.9', '3.10']
-        wagtail-version: ['2.11', '2.12', '2.13', '2.14', '2.15']
+        wagtail-version: ['2.11', '2.12', '2.13', '2.14', '2.15', '2.16']
```

Comment:
let's drop anything below 2.14 - https://endoflife.date/wagtail

Reply:
I updated the PR, and removed support for django <= 3.0 as 3.0 is the first django version supported by wagtail 2.14 https://docs.wagtail.org/en/latest/releases/2.14.html#removed-support-for-django-2-2